config/data/dataset/iwcd_cnn_short.yaml

```diff
@@ -1,4 +1,4 @@
-# @package _group_
 _target_: watchmal.dataset.cnn_mpmt.cnn_mpmt_dataset.CNNmPMTDataset
 mpmt_positions_file: /data/WatChMaL/data/IWCDshort_mPMT_image_positions.npz
-collapse_arrays: False
\ No newline at end of file
+collapse_arrays: False
+pad: True
```

Comment:
padding now on by default? 

update README.md with this and other options?

Reply:
I don't think the readme is the right place to document defaults. These config files are not exactly defaults anyway, you choose which files to load. Hydra config files are intended to provide a simple readable (i.e self-documenting) list of config options, so I think it's counterproductive to copy all these into the readme when every run will choose a different set of config files. The actual config is generally spread across many composed config files. So I think the readme should describe how to choose the different config files and how to modify the options. It should be more thoroughly reviewed in a later PR, but for now it just has an example of a config file which shouldn't really be considered "default".